src/pages/RepoPage/FailedTestsTab/FailedTestsTable/FailedTestsTable.tsx

```diff
@@ -0,0 +1,253 @@
+  testName: string
+  lastDuration: number
+  failureRate: number
+  commitsFailed: number
+  lastRun: string
+}
+
+const columnHelper = createColumnHelper<FailedTestsColumns>()
+
+const columns = [
+  columnHelper.accessor('testName', {
+    header: 'Test name',
+    cell: (info) => info.renderValue(),
+  }),
+  columnHelper.accessor('lastDuration', {
+    header: 'Last duration',
+    cell: (info) => info.renderValue(),
+  }),
+  columnHelper.accessor('failureRate', {
+    header: 'Failure rate',
+    cell: (info) => info.renderValue(),
+  }),
+  columnHelper.accessor('commitsFailed', {
+    header: 'Commits failed',
+    cell: (info) => info.renderValue(),
+  }),
+  columnHelper.accessor('lastRun', {
+    header: 'Last run',
+    cell: (info) => info.renderValue(),
+  }),
+]
+
+const FailedTestsTable = () => {
+  // const { ref, inView } = useInView()
+  const [sorting, setSorting] = useState<SortingState>([
+    {
+      id: 'failureRate',
+      desc: true,
+    },
+  ])
+  // const { owner } = useParams<{ owner: string }>()
+
+  // const {
+  //   data: reposData,
+  //   fetchNextPage,
+  //   hasNextPage,
+  //   isFetching,
+  //   isFetchingNextPage,
+  // } = useReposTeam({
+  //   activated: false,
+  //   sortItem: getSortingOption(sorting),
+  //   owner,
+  // })
+
+  // const tableData = useMemo(() => {
+  //   const data = reposData?.pages?.map((page) => page?.repos).flat()
+  //   return data ?? []
+  // }, [reposData?.pages])
+
+  const table = useReactTable({
+    columns,
+    data: [
+      {
+        testName: 'blah',
+        lastDuration: 123,
+        failureRate: 1,
+        commitsFailed: 4,
+        lastRun: '2021-01-01',
+      },
+      {
+        testName: 'cool',
+        lastDuration: 123,
+        failureRate: 2,
+        commitsFailed: 3,
+        lastRun: '2022-01-01',
+      },
+      {
+        testName: 'rad guy',
+        lastDuration: 123,
+        failureRate: 3,
+        commitsFailed: 2,
+        lastRun: '2023-01-01',
+      },
+      {
+        testName: 'awww ok',
+        lastDuration: 123,
+        failureRate: 4,
+        commitsFailed: 1,
+        lastRun: '2024-01-01',
+      },
+    ],
+    state: {
+      sorting,
+    },
+    onSortingChange: setSorting,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+  })
+
+  // useEffect(() => {
+  //   if (inView && hasNextPage) {
+  //     fetchNextPage()
+  //   }
+  // }, [fetchNextPage, inView, hasNextPage])
+
+  const isLoading = false
+
+  return (
+    <div className="tableui">
+      <table>
+        <colgroup>
+          <col className="w-full @sm/table:w-5/12" />
+          <col className="@sm/table:w-1/12" />
+          <col className="@sm/table:w-1/12" />
+          <col className="@sm/table:w-1/12" />
+          <col className="@sm/table:w-1/12" />
+        </colgroup>
+        <thead>
+          {table.getHeaderGroups().map((headerGroup) => (
+            <tr key={headerGroup.id}>
```

Comment:
The data in the column look shifted to the right, do you mind aligning that with the design please? 

<img width="652" alt="Screenshot 2024-07-23 at 11 47 37 AM" src="https://github.com/user-attachments/assets/eeec3ed9-aefe-4e7c-9bd4-55a8672ff4a7">

![Screenshot 2024-07-23 at 11 47 18 AM](https://github.com/user-attachments/assets/2ce36c4f-29e3-4886-8d82-16d294f3f716)

Reply:
Updated in https://github.com/codecov/gazebo/pull/3042/commits/c4a89ae4cab97632d503a259e0a9c68975c56d13